WalletWasabi.Fluent/Extensions/AvaloniaExtensions.cs

```diff
@@ -6,13 +6,13 @@ namespace WalletWasabi.Fluent.Extensions;
 
 public static class AvaloniaExtensions
 {
-	public static IObservable<EventPattern<TEventArgs>> OnEvent<TEventArgs>(
```

Comment:
Why change of formatting ?

Reply:
Others find this format easier to read:

https://github.com/zkSNACKs/WalletWasabi/pull/8916#discussion_r961231260
https://github.com/zkSNACKs/WalletWasabi/pull/8916#discussion_r969480990

+ I applied the Boy Scout Rule ;)

Reply:
Restored :)